packages/palette/src/elements/Cards/MediumCard.tsx

```diff
@@ -1,10 +1,50 @@
 import React from "react"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { MediumCardProps } from "./MediumCard.shared"
 
 /**
  * `MediumCard` is a card with one image one tall image, and text for title and subtitle
  * at the bottom.
  */
-export const MediumCard: React.FC<MediumCardProps> = ({}) => {
-  return null
+export const MediumCard: React.FC<MediumCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag
+}) => {
+  return (
+    <Box width="280px" height="370px" position="relative">
+      <Flex
+        width="100%"
+        height="100%"
+        position="absolute"
+        top="0"
+        borderRadius="4px"
+        overflow="hidden"
+      >
+        <Image src={image} alt={title} height="100%" width="auto" />
+        <Box
+          style={{
+            background: "linear-gradient(rgba(0, 0, 0, 0), rgba(0,0,0,1))",
+            opacity: 0.15
+          }}
+          position="absolute" width="100%" height="100%"
+        />
+        <Box position="absolute" bottom="15px" left="15px">
```

Comment:
Seems odd to me to step outside the spacing scale — would ask design why?

Reply:
I believe the small card has 10px and the large card has 20px, and so medium falls in the middle which is outside the spacing scale, so it kinda makes sense.

Reply:
We shouldn't ever really step outside of the spacing scale as one-offs like this add up. Rounding down or up would probably be fine?

Reply:
we rounded down to 10px on the native version, for what it's worth https://github.com/artsy/palette/pull/717/files/e44bd4c4bfee5c2a0720fc5c54c8934b9831d8f9#diff-8af20cc56832db8da1d1fc1c57c71860

Reply:
Conferred with @tinn on this, seems like we're already using 15px in other places, it just isn't in palette yet. Not sure how to resolve or what that means for this PR though, when design is saying 10px is too tight for this card. Would setting 15px be ok for now and we go back and clean it up when that spacing makes its way into palette?

Reply:
We changed it to 15 for ios too. Tin said small cards get 10 (so 1), medium get 15, large get 20 (so 2).
https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Cards/MediumCard.ios.tsx#L62